src/client/componentTypeMappings.js

```diff
@@ -8,27 +8,36 @@ import FileInput from "./ui/inputs/FileInput";
 import NumericInput from "./ui/inputs/NumericInput";
 import StringInput from "./ui/inputs/StringInput";
 import Vector3Input from "./ui/inputs/Vector3Input";
+import SelectInput from "./ui/inputs/SelectInput";
 
 /* eslint react/display-name: 0 */
 const componentTypeMappings = new Map([
-  [types.color, (value, isValid, onChange) => <ColorInput value={value} onChange={onChange} />],
-  [types.string, (value, isValid, onChange) => <StringInput value={value} onChange={onChange} />],
+  [types.color, (component, prop, value, isValid, onChange) => <ColorInput value={value} onChange={onChange} />],
```

Comment:
minor nit, but i'd call those `props`

Reply:
Yeah `prop` is a bit unclear. I think it should be `propDef` or `propSchema` though. I's a single entry from the component's schema.